frontend/test/metabase/scenarios/admin/settings/settings.cy.spec.js

```diff
@@ -356,6 +356,26 @@ describe("scenarios > admin > settings", () => {
       cy.findByText("Sorry, something went wrong. Please try again.");
     });
 
+    it("should send a test email for a valid SMTP configuration", () => {
+      // We must clear maildev inbox before each run - this will be extracted and automated
+      cy.request("DELETE", "http://localhost:80/email/all");
+      cy.request("PUT", "/api/setting", {
+        "email-smtp-host": "localhost",
```

Comment:
`localhost` ? isn't it `maildev` ? Or have the ports been exposed to where Cypress is being run from?
I would normally advise against munging all ports onto the host - it's easier to understand different hostname references.

Reply:
It is actually a `localhost`. I am not sure why or where does `maildev` in ENVs came from. @paoliniluis do you remember?